teamprojekt_competition_server/server/databases.py

```diff
@@ -1,8 +1,11 @@
 """Databases"""
 
 import sqlite3
+from enum import Enum
+from datetime import datetime
 
 GAME_DB_NAME = "game"
```

Comment:
I am still confused by this, we fix the name of the database but need to create a new object for acsessing the same database everytime ?

Reply:
changed it into a module